docs/useful_links.rst

```diff
@@ -0,0 +1,19 @@
+Useful links
+============
+
+**Related projects, that build abstractions on top of virtualenv**
+
+* `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`_ - a useful set of scripts for creating and deleting virtual environments
```

Comment:
Please restore these links to pypi. The pypi contains links to the documentation, code base etc.

Reply:
Add back pew.

Reply:
I had initially written the reasonings for this in my initial spec. Apologies, I thought you had approved those.

> Please restore these links to pypi. The pypi contains links to the documentation, code base etc.

PyPI is not an interface that a lot of people visit everyday.

> Add back pew.

`pew`'s last release/commit is from 2019. How is it relevant?

Reply:
I had described this in my initial spec on the revamp issue

> Please restore these links to pypi. The pypi contains links to the documentation, code base etc.

PyPI isn't a interface that is visited a lot, I am not sure why anything else over the official docs is preferred here



> Add back pew.

Pew's last commit/release was in 2019, I don't see how it's relevant. There are probably a bunch of other tools that are used in the ecosystem today that deserves a name here

Reply:
Not everything needs a new release to be relevant. pew is even more popular than nox, so should we remove that too; see https://pepy.tech/project/nox vs https://pepy.tech/project/pew